go/enclave/gas/gas.go

```diff
@@ -0,0 +1,54 @@
+package gas
```

Comment:
pls add a readme to this package describing the responsibilities and the behaviour